src/components/ContentSidebar/ActivityFeed/task/Task.js

```diff
@@ -69,6 +68,14 @@ class Task extends React.Component<Props> {
             getMentionWithQuery,
             mentionSelectorContacts,
         } = this.props;
+
+        const isCommentAuthor = currentUser && created_by && created_by.id === currentUser.id;
```

Comment:
I'm curious if this can be used elsewhere, in a util method that can be used with comments. `isItemOwner()`, or something like that

Reply:
In theory this is a good idea but we never have the data in the right place. A more generic function would `const isCommentAuthor = isSameUser(currentUser,created_by);`